### PR TITLE
Some edits on key management and  some questions

### PR DESCRIPTION
**Pull request by jhdalek55**
Can we get someone to go read through this relatively short page? The second paragraph in the online vs. offline section seems a bit contradictory. It first says "Similarly, on the Image repository, the Timestamp and Snapshot metadata SHOULD be signed using online keys so fresh metadata can be instantly generated by an automated process." But,  then the second sentence says that Offline keys can be used. I did edit these sentences to eliminate some repetition of words and phrases, but this contradiction was in the original wording as well--phrased differently but same meaning.

### CODE REVIEW

**Comment by trishankatdatadog** [comment]
Yeah, I agree, it's confusing.

We should say: look, there's two options for signing timestamp+snapshot metadata on the Image repository: online vs offline. (The pros vs cons for each option looks correct to me.) What we need to say is that for most use cases, you probably want online, but if you want stronger security guarantees, look at offline.

Does this make sense, Lois?

**Comment by jhdalek55** [comment]
Ok. I'm going to commit some new text.

File: key_management.md [hunk 1]
```diff
@@ -16,18 +16,20 @@ On both the Director and the Image repository, the OEM maintains the keys to the
...
 
-The keys to all other roles (Root, Targets, and all delegations, which includes suppliers' keys) on the Image repository SHOULD be kept offline. This is because these metadata are expected to be updated relatively infrequently and so that a repository compromise does not immediately affect full verification ECUs. It does not matter where an offline key is stored (e.g., in a Hardware Security Module, YubiKey, or a USB stick in a safe deposit box), as long as the key is not accessible from the repository. Each key SHOULD be kept separate from others, so that a compromise of one does not affect them all.
+In most use cases, the online option may be best, but if you want stronger security guarantees, consider using  online for the Timestamp and Snapshot roles.
```
Comment (trishankatdatadog) [inline key_management.md]:
```suggestion
In most use cases, the online option may be best, but if stronger security guarantees are desired, consider using  the offline option instead for the Timestamp and Snapshot roles.
```

**Comment by jhdalek55** [comment]
Of course...that was dumb mistake. I accepted the change. Thanks.

**Comment by trishankatdatadog** [comment]
@jhdalek55 no worries, these things happen, that's what reviews are for!

**Pull request merged**